docs/documentation/getting-started/deployment/manual/platform-setup.md

```diff
@@ -131,7 +140,13 @@ You can switch between the clusters easily using:
       name: cs-istiocontrolplane
     spec:
       meshConfig:
+        defaultConfig:
+          proxyMetadata:
+            ISTIO_META_DNS_AUTO_ALLOCATE: "true"
+            ISTIO_META_DNS_CAPTURE: "true"
         accessLogFile: /dev/stdout
+        #outboundTrafficPolicy:
```

Comment:
Shouldn't this be removed as it is commented anyways?

Reply:
was left for discussion, but sure, if it stays _disabled_ as it is untested and there is no evidence for a real requirement, it can and should be removed

Reply:
-> removed